scripts/links.js

```diff
@@ -0,0 +1,41 @@
+import jQuery from 'jquery';
+import { jsdom } from 'jsdom';
+import path from 'path';
+import replaceExtension from 'replace-ext';
+import isAbsoluteUrl from 'is-absolute-url';
+
+const $ = jQuery(jsdom().defaultView);
+
+/**
+ * Markdown doesn't render absolute links to markdown files (see https://github.com/mkdocs/mkdocs/issues/1172)
+ * 
+ * @param {String} html 
+ */
+export function reassignMarkdownLink (html) {
+  const jqHTML = $(html);
+  const anchorTags = jqHTML.find('a');
+  anchorTags.each((index, tag) => {
+    const anchorTag = $(tag);
+    const href = anchorTag.attr('href');
+
+    if (href && !isAbsoluteUrl(href)) {
+      const ext = path.extname(href);
+      if (ext === '.md') {
+        anchorTag.attr('href', `${replaceExtension(href, '')}/index.html`);
+      }
+    }
+  });
+  return jqHTML.html();
+}
+
+/**
+ * Takes a document, parses out the body and then adds fenced code to the body
```

Comment:
description seems wrong, this has nothing to do with fenced code, right?

Reply:
Yep.